### DISCUSSION TITLE
Can not access table with [owner].[tablename] name format on MS SQL

### DISCUSSION THREAD

**Discussion by pinkchink** [sequelize/sequelize#14511] [Help & Questions] [answered]
Hello

I have table names in owner.tablename format on MS SQL server (forced by db server owner)

If i use sequelize.query("SELECT * FROM [owner].[tablename]  => everything works ok

but if i use  model.findAll(); with explicitly set tablename in the same format  => sequelize translates it to [owner.tablename] and the query obviously fails 

Is there any workaround or specific syntax i can use to fix this?

my code and response:

import { Sequelize, DataTypes } from "sequelize";
var sequelize = new Sequelize(
  `db`,
  `user`,
  `pass`,
  {
    dialect: "mssql",
    host: `ipadress`,

    dialectOptions: {
      options: {
        useUTC: false,
        dateFirst: 1,
      },
    },
  }
);

const model = sequelize.define("[owner].[tablename]", {
    code: {
        type: DataTypes.INTEGER,
        allowNull: false,
        autoIncrement: true,
        primaryKey: true,
    },
    email: {
        type: DataTypes.STRING,     
    },
},
{  freezeTableName: true, tableName: `[owner].[tablename]`}

await model.findAll();

error:
sql: 'SELECT [code], [email], [createdAt], [updatedAt] FROM [owner.tablename] AS [[owner.tablename]];',
 original: RequestError: Invalid object name 'owner.tablename'.

**Comment by ephys** [comment, marked as answer]
You can use [the schema option](https://sequelize.org/api/v7/interfaces/modeloptions#schema)

**Reply by pinkchink** [reply]
thank you that helped
